core/arch/x86/opcode.h

```diff
@@ -1318,6 +1318,12 @@ enum {
     /* 1155 */ OP_ktestb,   /**< IA-32/AMD64 AVX-512 ktestd opcode. */
     /* 1156 */ OP_ktestq,   /**< IA-32/AMD64 AVX-512 ktestd opcode. */
     /* 1157 */ OP_ktestd,   /**< IA-32/AMD64 AVX-512 ktestd opcode. */
+    /* 1158 */ OP_vmovdqa32, /**< IA-32/AMD64 AVX-512 ktestd opcode. */
```

Comment:
Exactly these instructions will go upstream shortly, along with vmovss, vmovsd. Please pick instructions from i#1312, mark with (WIP), maybe (WIP JivanH) is better... There are still hundreds of instructions left, plenty of work to pick from!